ui/lib/components/Bar/Bar.module.less

```diff
@@ -12,6 +12,7 @@
   flex-direction: row;
   align-items: center;
   min-height: 1em;
+  height: 100%;
```

Comment:
This might be a breaking change, since `height: 100%` varies depends on its parent container. Do we really need this style rule?